python/ml4ir/base/model/relevance_model.py

```diff
@@ -390,27 +561,37 @@ def save(
         pad_sequence: bool = False,
     ):
         """
-        Save tf.keras model to models_dir
+        Save the RelevanceModel as a tensorflow SavedModel to the `models_dir`
 
-        Two different serving signatures currently used to save the model
-            default: default keras model without any pre/post processing wrapper
-            tfrecord: serving signature that allows keras model to be served using TFRecord proto messages.
-                      Allows definition of custom pre/post processing logic
+        There are two different serving signatures currently used to save the model:
+        * `default`: default keras model without any pre/post processing wrapper
```

Comment:
insert a blank line between lines 566 & 567 for the list formatting

Reply:
Done.